packages/design-tokens/src/tokens/functional/components/pricing-cards/pricing-cards.json

```diff
@@ -0,0 +1,42 @@
+{
+  "brand": {
+    "PricingOptions": {
+      "items": {
+        "gap": {
+          "value": "var(--base-size-24)"
+        },
+        "container": {
+          "padding": {
+            "inline": {
+              "value": "0"
```

Comment:
Does this need to be a token if the value is 0 by default? I.e. if end-user wants to customise it, could they pass a class into that container? Is it even something we want users to customise?